tests/test_vrt.py

```diff
@@ -29,19 +29,28 @@ def test_vrt_schema(self):
     def test_cvss_v3_schema(self):
         self.validate_schema(utils.CVSS_V3_SCHEMA_FILENAME, utils.CVSS_V3_MAPPING_FILENAME)
 
+    def test_cwe_schema(self):
+        self.validate_schema(utils.CWE_SCHEMA_FILENAME, utils.CWE_MAPPING_FILENAME)
+
     def test_all_vrt_ids_have_mapping(self):
         vrt = utils.get_json(utils.VRT_FILENAME)
-        mapping = utils.get_json(utils.CVSS_V3_MAPPING_FILENAME)
-        keyed_mapping = utils.key_by_id(mapping['content'])
-        for vrt_id_list in utils.all_id_lists(vrt, include_internal=False):
-            self.assertTrue(utils.has_mapping(keyed_mapping, vrt_id_list, 'cvss_v3'),
-                            'no cvss_v3 mapping for ' + '.'.join(vrt_id_list))
+        mappings = {}
+        mappings["cvss_v3"] = utils.get_json(utils.CVSS_V3_MAPPING_FILENAME)
+        mappings["cwe"] = utils.get_json(utils.CWE_MAPPING_FILENAME)
+        for key, mapping in mappings.iteritems():
+            keyed_mapping = utils.key_by_id(mapping['content'])
+            for vrt_id_list in utils.all_id_lists(vrt, include_internal=False):
+                self.assertTrue(utils.has_mapping(keyed_mapping, vrt_id_list, key),
+                                'no ' + key + ' mapping for ' + '.'.join(vrt_id_list))
```

Comment:
Running off the general principle of "less assertions per test case is better", I have a vague preference for something like:
```python
def all_vrt_ids_have_mapping(self, mappping_filename, key):
    ...

def test_all_vrt_ids_have_cvss_mapping(self):
    self.all_vrt_ids_have_mapping(utils.CVSS_V3_MAPPING_FILENAME, 'cvss_v3')

def test_all_vrt_ids_have_cwe_mapping(self):
    self.all_vrt_ids_have_mapping(utils.CWE_MAPPING_FILENAME, 'cwe')
```

(methods starting with `test_` are the test cases, everything else is a helper method). But these tests already have so many assertions that it doesn't really matter.